gdal/port/cpl_vsisimple.cpp

```diff
@@ -1158,36 +1098,7 @@ void VSIFree( void * pData )
 
 void* VSIMallocAligned( size_t nAlignment, size_t nSize )
```

Comment:
I don't like how the argument order is swapped here (and they're the same type). Seems error prone.
Unfortunately as it's in gdal we can't change it. How it does mean that if someone replaces `_aligned_malloc` with `VSIMallocAligned` they'll need to reverse their argument order.